packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -66,6 +66,23 @@ $footer-height: 70px;
 	margin-bottom: 0;
 }
 
+.font-library-modal__fonts-title {
```

Comment:
Not a blocker from me, but in general it would be nice to move away from custom CSS in favor of the base components with their default styles. I assume this would require a larger refactor?

Reply:
Similar to these styles is the label for the `BaseControl` component. However, we cannot use the label as a heading element.

![image](https://github.com/WordPress/gutenberg/assets/54422211/1eea1fbb-2636-4867-b6ae-c8df49e10d36)

Personally, I think it's fine as is for now.

Reply:
It would be nice to have a heading style for that appearance that can be used in more cases.